tests/python/relax/test_op_nn.py

```diff
@@ -122,7 +122,7 @@ def test_linear_unit_infer_struct_info_wrong_input_type():
         bb.normalize(relax.op.nn.silu(x1))
 
 
-def test_softmax_infer_struct_info():
+def test_softmax_log_softmax_infer_struct_info():
```

Comment:
Please add `log_softmax` and `nll_loss` to the getter correctness tests.
https://github.com/mlc-ai/relax/blob/338ad67857fd6d552e13290df76e45864b1767f9/tests/python/relax/test_op_nn.py#L31-L32

Reply:
Also please check the collapse_sum side. I forgot to pay attention to that last time.